product_docs/docs/biganimal/release/using_cluster/pgd_cli_ba.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: PGD CLI on BigAnimal
+navTitle: PGD CLI on BigAnimal
+deepToC: true
+---
+
+When running a distributed high-availability cluster (DHA) on BigAnimal, you can use the [PGD CLI](../../../pgd/latest/cli/) to manage cluster operations, such as switching over write leaders, performing cluster health checks, and viewing various details about nodes, groups, or other aspects of the cluster.
+
+## Installing the PGD CLI 
+
+To [install the PGD CLI](../../../pgd/latest/cli/installing_cli/), replace `<your-token>` with your EDB subscription token in the following command for Debian and Ubuntu machines:
+
+```bash
+curl -1sLf 'https://downloads.enterprisedb.com/&lt;your-token&gt;/postgres_distributed/setup.deb.sh' | sudo -E bash
+sudo apt-get install edb-pgd5-cli
+```
+
+or in this command for RHEL, Rocky, AlmaLinux, or Oracle Linux machines:
+
+```bash
+curl -1sLf 'https://downloads.enterprisedb.com/&lt;your-token&gt;/postgres_distributed/setup.rpm.sh' | sudo -E bash
+sudo yum install edb-pgd5-cli
+```
+
+## Connecting to your BigAnimal cluster
+
+### Discovering your database connection string
+
+To connect to your DHA BigAnimal cluster via the PGD CLI, you need to [discover the database connection string](../../../pgd/latest/cli/discover_connections/) from your BigAnimal console:
```

Comment:
@jpe442 we don't use the DHA acronym anywhere. It's always PGD

Reply:
You're right we don't use the DHA acronym, but we do not call them PGD in BA docs as often: for instance: http://localhost:8000/biganimal/latest/overview/02_high_availability/distributed_highavailability/. This is because they are not called PGD clusters in the BA UI, they are called distributed high-availability.